lib/gh_inspector/evidence.rb

```diff
@@ -2,24 +2,32 @@
 require 'time'
 
 module GhInspector
+  NUMBER_OF_ISSUES_INLINE = 3
+
   # The default user interface for the inspector, its public API should be
   # considered the protocol for other classes wanting to provide a user interface.
   #
   # Your custom objects will be verified at runtime that they conform to the protocol.
   #
-  # You can see the default implmentation at
+  # You can see the default implementation at
```

Comment:
Typo fixed [here](https://github.com/orta/gh_inspector/commit/4b6ed2d567bbc268fc042034c9147c255a1d7702#diff-04c6e90faac2675aa89e2176d2eec7d8R60), alongside the other changes.